src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -95,36 +94,33 @@ export default function FundingForm({
 
   const [fundingForm] = Form.useForm<FundingFormFields>()
 
+  // Load redux state (will be empty in create flow)
+  const { fundAccessConstraints, fundingCycleData, payoutGroupedSplits } =
+    useAppSelector(state => state.editingV2Project)
+  const fundAccessConstraint =
+    getDefaultFundAccessConstraint<SerializedV2FundAccessConstraint>(
+      fundAccessConstraints,
+    )
+
   const [splits, setSplits] = useState<Split[]>([])
   // Must differentiate between splits loaded from redux and
   // ones just added to be able to still edit splits you've
   // added with a lockedUntil
   const [editingSplits, setEditingSplits] = useState<Split[]>([])
 
-  const [target, setTarget] = useState<string | undefined>()
+  const [distributionLimit, setDistributionLimit] = useState<
+    string | undefined
+  >(fundAccessConstraint?.distributionLimit ?? '0')
```

Comment:
`fundAccessConstraint?.distributionLimit` won't be undefined if reconfiging or coming back to a saved create flow instance, but yeah since it's being set in `resetProjectForm` which is being called in a `useLayoutEffect` this is indeed unnecessary